internal/innosat/telemetry.go

```diff
@@ -51,6 +51,15 @@ func (header *TMHeader) IsTransparentData() bool {
 	return header.ServiceType == 128 && header.ServiceSubType == 25
 }
 
+// IsTCVerification returns true if payload contains TC verification data
+func (header *TMHeader) IsTCVerification() bool {
+	return header.ServiceType == TelecommandVerification &&
+		(header.ServiceSubType == 1 ||
```

Comment:
And use the same consts here, because now you have the magic numbers in two places and thus I feel even stronger about it.